azure-cli/deploy.services.ps1

```diff
@@ -152,7 +153,7 @@ $logAnalyticsKey = Get-LogAnalyticsKey `
   -LogAnalyticsName $logAnalyticsName `
   -ResourceGroup $resourceGroupName
 
-$instrumentationKey = Initialize-ApplicationInsights `
+$connectionString = Initialize-ApplicationInsights `
```

Comment:
maybe choose a different name in main script, to avoid trouble down the road

Reply:
aiConnectionString ?